com.vscodium.codium-insiders.yaml

```diff
@@ -0,0 +1,144 @@
+id: com.vscodium.codium-insiders
+sdk: org.freedesktop.Sdk
+runtime: org.freedesktop.Sdk
+runtime-version: '23.08'
+command: codium-insiders
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=wayland
```

Comment:
Does it support native wayland or x11 and xwayland only? If x11/xwayland only remove wayland for now, otherwise change x11 to fallback-x11. You can add them if wayland is enabled later.

Reply:
this wasn't addressed.